_includes/specials/policy/en/09-proposals.md

```diff
@@ -0,0 +1,113 @@
+[Last week’s post][policy08] described anchor outputs and CPFP carve
+out, ensuring either channel party can fee-bump their shared
+commitment transactions without requiring collaboration. This approach
+still contains a few drawbacks: channel funds are tied up to create
+anchor outputs, commitment transaction feerates typically overpay
+to ensure they meet mempool minimum feerates, and CPFP carve out only
+allows one extra descendant. Anchor outputs do not ensure the same
+ability to fee-bump for transactions shared between more than two
+parties, such as CoinJoins or multi-party contracting protocols. This
+post explores current efforts to address these and other limitations.
+
+[Package relay][topic package relay] includes p2p protocol and policy
+changes to enable the transport and validation of groups of
+transactions. It would allow a commitment transaction to be fee-bumped
+by a child even if a mempool’s minimum feerate exceeds that of the
+commitment transaction. Additionally, _Package RBF_ would allow
```

Comment:
```suggestion
transactions. It would allow a commitment transaction to be fee-bumped
by a child even if the commitment transaction does not meet a mempool’s minimum feerate. 
Additionally, _Package RBF_ would allow
```